server/socketio/connection.js

```diff
@@ -28,4 +31,23 @@ global.io.on("connection", (socket) => {
             socket.join(conversationId);
         }
     });
+
+    const authToken = socket.handshake.auth.token;
+    const user = jwt.verify(authToken, process.env.TOKEN_SECRET);
```

Comment:
Does this need to happen twice? What happens in the above call to `global.io.use`?